src/Xamarin.Android.Build.Tasks/Tasks/ConvertCustomView.cs

```diff
@@ -79,24 +79,23 @@ public override bool RunTask ()
 					}
 				}
 			}
-			var output = new List<ITaskItem> ();
+			var output = new Dictionary<string, ITaskItem> ();
```

Comment:
Minor, but could maybe be:
```suggestion
			var output = new Dictionary<string, ITaskItem> (processed.Count);
```